### PR TITLE
test-infra

### PR DESCRIPTION
**Pull request by jburel**
Use new test infra

Make sure travis is green

### CODE REVIEW

File: .omeroci/app-config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+set -e
+set -u
+OMERO_VENV=${OMERO_VENT:-/opt/omero/web/venv}
+OMERO_DIST=${OMERO_DIST:-/opt/omero/web/OMERO.web}
+${OMERO_VENV}/bin/python \
+    $OMERO_DIST/bin/omero config append omero.web.apps '"omero_webtest"'
```
Comment (joshmoore) [inline .omeroci/app-config]: Either as part of https://github.com/openmicroscopy/omero-test-infra/pull/5 or as a follow-up we should have the correct `python` as well as `omero` on the `$PATH` so this boilerplate is not needed.

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
+  - docker
 
 before_install:
-  - pip install restructuredtext_lint
-  - pip install flake8 pycodestyle pep8-naming
+  - git clone git://github.com/openmicroscopy/omero-test-infra .omero
 
 script:
-  - flake8 -v .
-  - rst-lint README.rst
```
Comment (sbesson) [inline .travis.yml:16]: Are the `flake8` and `rst-lint` steps done in omero-test-infra?

Comment (joshmoore) [reply .travis.yml:16]: yes: https://github.com/openmicroscopy/omero-test-infra/blob/master/py-setup

Comment (jburel) [reply .travis.yml:16]: yes
I have also introduced the ability to exclude some files/repo by using ``app-check``

File: .omeroci/app-config (hunk 1 above)
Comment (sbesson) [reply .omeroci/app-config]: @jburel have you tried collapsing these 4 lines to `omero config append omero.web.apps '"omero_webtest"'`?

Comment (jburel) [reply .omeroci/app-config]: As indicated on another PR, the plan is to remove the default config to test-infra

**Review by sbesson — APPROVED** [review]
Barring the discussion in https://github.com/openmicroscopy/omero-webtest/pull/26#discussion_r164351747, the changes here are in agreement with the strategy used for all downstream repositories.

All checks previously performed  by Travis are now covered by omero-test-infra.

File: .omeroci/app-config (hunk 1 above)
Comment (sbesson) [reply .omeroci/app-config]: Understood, can you cross-link to the reference PR or even card recording the series of step?

Comment (jburel) [reply .omeroci/app-config]: https://github.com/ome/omero-fpbioimage/pull/19

**Comment by jburel** [comment]
I will revisit this PR when https://github.com/openmicroscopy/omero-test-infra/pull/3 is merged and the modification mentioned above integrated

**Comment by joshmoore** [comment]
Green (and nicely flake8'd)

**Pull request merged**